src/vm/codeman.cpp

```diff
@@ -5062,8 +5143,54 @@ PCODE ExecutionManager::getNextJumpStub(MethodDesc* pMD, PCODE target,
         pJumpStubCache->m_Table.Add(entry);
     }
 
-    curBlock->m_used++;
+    curBlock->m_used++;    // record that we have used up one more jumpStub in the block
+
+    // Every time we create a new jumpStub thunk one of these counters is incremented
+    if (isLCG)
+    {
+        // Increment counter of LCG unique jump stubs
+        m_LCG_JumpStubUnique++;
+    }
+    else
+    {
+        // Increment counter of normal unique jump stubs 
+        m_normal_JumpStubUnique++;
+    }
 
+    // Is the 'curBlock' now completely full?
+    if (curBlock->m_used == curBlock->m_allocated)
+    {
+        if (isLCG)
+        {
+            // Increment counter of LCG jump stub blocks that are full
+            m_LCG_JumpStubBlockFullCount++;
+
+            // Log this "LCG JumpStubBlock filled" along with the four counter values
+            STRESS_LOG4(LF_JIT, LL_INFO100, "LCG JumpStubBlock filled - (%u, %u, %u, %u)\n",
```

Comment:
I think this logging level is too low. JITing is level `LL_INFO10000`. I think this should be higher than that.